src/Surfnet/StepupMiddleware/ManagementBundle/Validator/IdentityProviderConfigurationValidator.php

```diff
@@ -27,22 +27,29 @@ public function validate(array $configuration, $propertyPath)
     {
         Assertion::isArray($configuration, 'invalid configuration format, must be an object', $propertyPath);
 
-        $acceptedProperties = [
+        $requiredProperties = [
```

Comment:
In my opinion, the newly chosen name is a step back in descriptiveness. The `use_pdp` property is not really required as it defaults to false when it is not configured. This also goes for the `ServiceProviderConfigurationValidator`.